superset-frontend/cypress-base/cypress/integration/sqllab/tabs.test.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+describe('SqlLab query tabs', () => {
+  beforeEach(() => {
+    cy.login();
+    cy.visit('/superset/sqllab');
+  });
+
+  it('allows you to create and close a tab', () => {
```

Comment:
I've rewritten the tests for SQL Lab Tabs to make them more re-runnable---so that if you have any backend persistent tabs, the tab names are still named correctly following the logics [here](https://github.com/apache/superset/blob/5a5ff99c372041e14621bc20015edd1b057cb417/superset-frontend/src/SqlLab/components/TabbedSqlEditors/index.jsx#L267-L277).

cc @cccs-Dustin @kgabryje